### PR TITLE
Remove AWS related Dependencies and Config

### PR DESCRIPTION
**Pull request by yccodr**
## What has been done?

- Remove AWS related Dependencies
- Remove AWS Cognito related components and code
- Remove AWS related config
- (Minor) Upgrade TS build target to `ES2022`

### CODE REVIEW

**Review by andy89923 — APPROVED** [review]
LGTM

File: frontend/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2022",
```
[the remark this answers is not in the document]
Comment (andy89923) [reply frontend/tsconfig.json:3]: > While the ECMAScript 2023 spec is very new, there is already [good support for these new array methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#browser_compatibility). Chrome 110, Safari 16.3,...

From [here](https://www.sonarsource.com/blog/es2023-new-array-copying-methods-javascript/).

- safari 16.3 was published in Nov 2023. 
- chrome 110 was published in Feb 2023.

[the remark this answers is not in the document]
Comment (andy89923) [reply frontend/tsconfig.json:3]: ES2022: https://www.w3schools.com/js/js_2022.asp.

[the remark this answers is not in the document]
Comment (yccodr) [reply frontend/tsconfig.json:3]: If you are talking about IE...
but it's dead. RIP 😉

**Review by ianchen0119 — APPROVED** [review]

**Pull request merged**